ndcube/ndcube.py

```diff
@@ -366,9 +371,109 @@ def extra_coords(self):
                     "value": self._extra_coords_wcs_axis[key]["value"]}
         return result
 
+    def all_world_coords(self, axes=None):
```

Comment:
I would change this to `(self, *axes)` the default would then be an empty tuple and not `None`.